recipes-ni/sysconfig-settings-ssh/sysconfig-settings-ssh_1.0.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "System configuration files for ssh"
+DESCRIPTION = "SSH configuration files for the National Instruments System Configuration subsystem."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
```

Comment:
I think I would prefer for `sysconfig-settings-ssh` to be a subpackage of the `sysconfig-settings` recipe, rather than living as its own recipe. That will make long-term maintenance a little easier.

Reply:
Done

Reply:
Should I also merge `sysconfig-settings-ui` package into this recipe?

Reply:
I think that would make sense to do; yeah.

Note that `sysconfig-settings-ui` does legitimately have an `niacctbase` dependency.

Reply:
Done